sdk/python/sawtooth_sdk/processor/core.py

```diff
@@ -13,17 +13,28 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+from concurrent.futures import CancelledError
+from concurrent.futures import TimeoutError as TOError
```

Comment:
Renaming things on import generally makes it harder to read the code.  If this conflicts with another TimeoutError, we could instead refer to it fully qualified concurrent.futures.TimeoutError.